ui/ducks/metamask/metamask.js

```diff
@@ -81,6 +82,26 @@ export default function reduceMetamask(state = initialState, action) {
         isUnlocked: false,
       };
 
+    case actionConstants.UPDATE_ACTIVE_NETWORK: {
+      const { networkConfigurationId, rpcUrl } = action.value;
+      return {
+        ...metamaskState,
+        provider: {
+          type: NETWORK_TYPES.RPC,
+          rpcUrl,
+          networkConfigurationId,
+        },
+      };
+    }
+
+    case actionConstants.SET_PROVIDER_TYPE:
```

Comment:
Is this being used anywhere?

Reply:
Similarly, the `UPDATE_ACTIVE_NETWORK` action appears to be new as well. Maybe this was reintroduced accidentally when resolving conflicts, I know Brad has been removing unused actions from here recently.

Reply:
removed both here: fixed here: https://github.com/MetaMask/metamask-extension/pull/17421/commits/ef2df393c9838e83875c78b10cc411b0cdb6c76c